aws/table_aws_ec2_ami.go

```diff
@@ -103,6 +103,11 @@ func tableAwsEc2Ami(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_STRING,
 				Default:     "self",
 			},
+			{
+				Name:        "imds_support",
+				Description: "Indicates that IMDSv2 is specified in the AMI",
```

Comment:
Should the column datatype be string? In my opinion, this should be boolean.

Reply:
@misraved According to https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/ec2@v1.73.0/types#Image, the value is based off of https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/ec2@v1.73.0/types#ImdsSupportValues, which is a string. In the future, it looks like there may be additional versions beside `2.0`.